src/scrabble.js

```diff
@@ -1,5 +1,131 @@
-function scrabble() {
-  // write code here
+// PLAN
+
+// Plain english
+
+// GOAL -
+// The purpose is to pass in a word via a function and that word should return a score depending on the word passed through. The letters should be assigned with points - the points should add together to give a total score.
+
+// For example,
+// A = 1
+// P = 3
+// P = 3
+// L = 1
+// E = 1
+// Total score should return a score of 9
+
+// However, if a white space is entered - ('') empty string = this should return 0
+// However, if a new line is entered /n   (" \t\n")         = should return 0
+// However, if null (nothing) is entered                              = should return 0
+
+// Pseudo code
+
+// 1. Create a function called scrabble
+// //  2. Assign each letter to a score for example, A=1
+// Write your logic here so:
+// if A, E, I, O, U, L, N, R, S, T then return 1
+// if D, G                         then return 2
+// if B, C, M, P                   then return 3
+// if F, H, V, W,                  then return 4
+// if K                            then return 5
+// if J, X                         then return 8
+// if Q, Z                         then return 10
+
+// 2. Need a loop that goes through each letter after the word is entered in the function, however as null is not a string, cannot use the word.length condition. Therefore have these conditons outside the loop.
+// If statements ensuring if:
+// - '' is entered should return 0
+// - \t\n is entered should return 0
+// - null should return 0
+
+// 5. Create a variable that keeps track of the score
+
+// 6. Create a loop to loop through each word entered
+// - if any word entered
+// add the totalScore to the scoreLetters
```

Comment:
Ohh that's very good to do plan for your exercise it is very big and clear plan good job I like it :)

Reply:
Thanks so much